src/main/java/net/dv8tion/jda/internal/requests/restaction/pagination/ThreadChannelPaginationActionImpl.java

```diff
@@ -21,11 +22,13 @@
 public class ThreadChannelPaginationActionImpl extends PaginationActionImpl<ThreadChannel, ThreadChannelPaginationAction> implements ThreadChannelPaginationAction
 {
     protected final IThreadContainer channel;
+    protected final boolean useID;
 
-    public ThreadChannelPaginationActionImpl(JDA api, Route.CompiledRoute route, IThreadContainer channel)
+    public ThreadChannelPaginationActionImpl(JDA api, Route.CompiledRoute route, IThreadContainer channel, boolean useID)
     {
-        super(api, route, 1, 100, 100);
+        super(api, route, 2, 100, 100);
```

Comment:
The minimum you can retrieve is 2? is this correct?

Reply:
It is 2 for private joined archived in my testing

Reply:
i changed it, because Minn mentioned it in the thread i created on Discord